scripts/ci/tags.yaml

```diff
@@ -97,7 +97,14 @@ posix:
   files:
     - zephyr/lib/posix/
 
-# cbprintf:
-#    files:
-#        - zephyr/lib/os/cbprintf*
-#        - zephyr/lib/posix
+suit:
```

Comment:
test-sdk-dfu is triggered by changes done in the following files/folders (test-spec.yaml):
```
"CI-suit-dfu-test":
  - "include/bl*"
  - "include/fprotect.h"
  - "include/dfu/*suit*"
  - "include/sdfw/sdfw_services/*"
  - "cmake/**/*"
  - "sysbuild/**/*"
  - "lib/dk_buttons_and_leds/**/*"
  - "subsys/dfu/*"
  - "subsys/nrf_rpc/**/*"
  - "subsys/nrf_security/**/*"
  - "subsys/sdfw_services/**/*"
  - "subsys/suit/**/*"
  - "samples/suit/**/*"
  - "subsys/mgmt/suitfu/**/*"
  - "tests/subsys/suit/**/*"

"CI-suit-dfu-test-extended":
  - "include/dfu/*suit*"
  - "include/sdfw/sdfw_services/*"
  - "subsys/sdfw_services/**/*"
  - "subsys/suit/**/*"
  - "samples/suit/**/*"
  - "subsys/mgmt/suitfu/**/*"
  - "tests/subsys/suit/**/*"
```
List above has been created during dependency analysis done by the team responsible for SUIT.
In my opinion the same files/folders shall be used to limit Twister tests/builds - if there is a plan to do so.

Reply:
Updated after evaluation of:
https://github.com/nrfconnect/sdk-nrf/blob/main/.github/test-spec.yml#L521-L545
https://github.com/nrfconnect/suit-generator/blob/ad6502a937d1e6e11824d3fb546f3d2027c5766a/.github/test-spec.yml#L1-L2
https://github.com/nrfconnect/suit-processor/blob/05089f4f7452ff50f09e2e2d776105c630381b1e/.github/test-spec.yml#L1-L2

Please note that the content is similar and syntax is different - the root path is NCS, all dependencies - from west modules - are stated at one place, tests and samples should not be included here, they are handled separately (but they will be build if changed)